automatic/nomacs/nomacs.nuspec

```diff
@@ -3,63 +3,58 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>nomacs</id>
-    <version>3.6.1.20170421</version>
+    <version>3.6.1</version>
     <title>nomacs Image Lounge</title>
     <owners>chocolatey</owners>
     <authors>Markus Diem, Stefan Fiel, Florian Kleber</authors>
     <licenseUrl>https://www.gnu.org/licenses/gpl.html</licenseUrl>
     <projectUrl>https://nomacs.org/</projectUrl>
     <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/bcd6e5bad67c9a360b1e9bc9a1ccfedfd4e57fdd/icons/nomacs.svg</iconUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>
+    <description><![CDATA[
 nomacs is a free image viewer for windows, linux, and mac systems, which is licensed under the GNU General Public License v3. nomacs is small, fast and able to handle the most common image formats including RAW images. Additionally it is possible to synchronize multiple viewers. A synchronization of viewers running on the same computer or via LAN is possible. It allows to compare images and spot the differences (e.g. schemes of architects to show the progress).
 
 ## Features
 
 - Image Viewing
-  - supports the most common image formats including RAW
-  - fast thumbnail preview
-  - zoomable grid preview of thumbnails
-  - display exif information
-  - framless view
-  - change transparency of windows
-  - display overview/histogram/file info
-  - file filtering/searching/sorting
-  - improved anti-aliasing
-  - hide menubar and toolbar
+- supports the most common image formats including RAW
+- fast thumbnail preview
+- zoomable grid preview of thumbnails
+- display exif information
+- framless view
+- change transparency of windows
+- display overview/histogram/file info
+- file filtering/searching/sorting
+- improved anti-aliasing
+- hide menubar and toolbar
 - Image Editing
-  - crop images
-  - resize images
-  - pseudo color function
-  - color adjustments (brightness, contrast, saturation …)
-  - multi page TIFF export
-  - rotate images
-  - drag and drop images
-  - automatically save screenshots
-  - delete/rename images
-  - create mosaic image
-  - set wallpaper (windows only)
+- crop images
+- resize images
+- pseudo color function
+- color adjustments (brightness, contrast, saturation â€¦)
```

Comment:
Seems like you have somehow gotten an invalid character here somehow. Would need to be fixed